lib/Runtime/PlatformAgnostic/Platform/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ set(PL_SOURCE_FILES
   Linux/UnicodeText.ICU.cpp
   Linux/NumbersUtility.cpp
   Linux/Thread.cpp
+  Linux/PerfTrace.cpp
```

Comment:
Not sure how dtrace could fit in the scenario but on OSX `Instruments` application does the job pretty well. Thus; for `instruments` not sure how this output would work. Actually it is already easy to identify the JS calls there. Only missing (default) functionality is naming the JS functions. We could left a todo for osx here though